elixir/apps/web/lib/web/live/resources/new.ex

```diff
@@ -134,12 +134,32 @@ defmodule Web.Resources.New do
                 disabled={is_nil(@form[:type].value)}
                 required
               />
+              <p
+                :if={
+                  @form[:type].value == :dns and
+                    is_binary(@form[:address].value) and
+                    @form[:address].value
+                    |> String.codepoints()
+                    |> Resources.map_resource_address() == :drop
+                }
+                class="flex items-center gap-2 text-sm leading-6 text-yellow-600 mt-2 w-full"
+              >
+                <.icon name="hero-exclamation-triangle" class="w-4 h-4" />
+                This is an advanced address format that is not supported by clients prior to v1.2.
```

Comment:
nit: slightly more clear this way imo

```suggestion
                This is an advanced address format that requires Client versions 1.2 and higher.
```

Reply:
There might be a slight usability issue here: if they enter an address that isn't "advanced", behavior is different for Clients less than 1.2 and greater than 1.2. In that case, the help text here is inaccurate. Is that the case?

Reply:
The migration from old to new is pretty easy. Maybe we should migrate everything first, ship the new clients but only allow the old format, migrating new entries on the fly before writing them to the DB.

Then we can give users time to upgrade their clients.

Reply:
> There might be a slight usability issue here: if they enter an address that isn't "advanced", behavior is different for Clients less than 1.2 and greater than 1.2. In that case, the help text here is inaccurate. Is that the case?

Not exactly sure how it's different? We take `**.example.com` and map that to `*.example.com` that would be the same, and if you do something 1.2 doesn't support - they simply won't see that resource.

Reply:
> if you do something 1.1 doesn't support - the client won't see that resource.

Ah that clears it up. I understand now. Nice!

Reply:
Let me make a quick suggestion to update the helptext